team.html

```diff
@@ -0,0 +1,551 @@
+							homelands and territories of Florida consisting of but not limited
+							to the Seminole Nation and other first peoples such as the Calusa,
+							Tobobaga, Timucua, Apalachee, Miccosukee, Mascogo, Taíno, amongst
+							others. ITLCT also acknowledge that we operate, and our leaders
+							also reside on the traditional homelands and territories of
+							Georgia consisting of the Cherokee Nation and other first peoples
+							such as but not limited to the Catawba, Muscogee (Creek),
+							Chickasaw, Shawnee, Oconee, Apalachicola, Chatot, amongst others.
+						</p>
+
+						<p>
+							The first peoples of Florida and Georgia who lived, worked,
+							nurtured the land, and shared knowledge were involuntarily
+							removed, faced difficult challenges and oppression, may still be
+							seeking recognition and reconciliation in some cases, but today,
+							they continue to shape their communities and the future in
+							positive ways. ITLCT recognizes this and vows to be humble and
+							respectful stewards of the lands, histories, and communities of
+							Florida and Georgia's first people and other global tribes,
+							nations, and communities as we engage in our work.
+						</p>
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<!-- team modal -->
+		<div class="modal-overlay team-modal hidden" id="team-modal">
+			<div class="modal-content">
+				<!-- button container -->
+				<div class="modal-btn-container">
+					<button
+						class="close-modal"
+						data-role="close-modal"
+						aria-label="open team member modal"
+					>
+						<svg
+							width="24"
+							height="24"
+							viewBox="0 0 24 24"
+							xmlns="http://www.w3.org/2000/svg"
+						>
+							<path
+								d="M1.7416 23.9993C1.29575 23.9993 0.849903 23.8293 0.509942 23.4894C-0.169981 22.8094 -0.169981 21.7067 0.509942 21.026L21.0267 0.509942C21.7067 -0.169981 22.8094 -0.169981 23.4901 0.509942C24.17 1.18986 24.17 2.29265 23.4901 2.97327L2.97257 23.4894C2.63261 23.8293 2.18745 23.9993 1.7416 23.9993Z"
+							/>
+							<path
+								d="M22.2577 24C21.8118 24 21.366 23.83 21.026 23.4901L0.509942 2.97327C-0.169981 2.29334 -0.169981 1.19056 0.509942 0.509942C1.18986 -0.169981 2.29265 -0.169981 2.97327 0.509942L23.4901 21.0267C24.17 21.7067 24.17 22.8094 23.4901 23.4901C23.1494 23.83 22.7035 24 22.2577 24Z"
+							/>
+						</svg>
+					</button>
+				</div>
+				<!-- member profile -->
+				<div class="member">
+					<!-- img -->
+					<div class="img">
+						<img
+							id="member-img"
+							src="/assets/images/pages/team/natasha-rabsatt.png"
+							alt=""
+						/>
+					</div>
+					<!-- text -->
+					<div class="text">
+						<!-- SM Bio -->
+						<div class="bio">
+							<h3 id="member-name"></h3>
+							<p class="role" id="member-role"></p>
+						</div>
+						<!-- connect CAT -->
+						<div class="member-cat">
+							<!-- email -->
+							<a id="member-email" href="" class="social-icon">
```

Comment:
Let's have these links open in a new window.

Reply:
Thank you for catching these! adjusted